pkg/config/system_probe.go

```diff
@@ -162,3 +164,10 @@ func InitSystemProbeConfig(cfg Config) {
 func join(pieces ...string) string {
 	return strings.Join(pieces, ".")
 }
+
+func suffixHostEtc(suffix string) string {
```

Comment:
As is, there would be no way to completely override these config values with a specific directory. Could we remove the leading `/` from the defaults, and check for a relative path before adding on the prefix?

Reply:
This just changes the default value doesn't it ? If it was specified in the config the specified value would be used

Reply:
Ah yes, you are correct. Apologies.